views/add-delivery-employee.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
For UI changes it's always good practice to include a screenshot in the PR Description